instaseis/gui/instaseis_gui.py

```diff
@@ -217,7 +217,24 @@ def plot_map(self):
 
         self.map.drawmapboundary(fill_color='#cccccc')
         if self.instaseis_db and self.instaseis_db.info.planet_radius < 5e6:
-            imfile = os.path.join(DATA, 'mola_texture_shifted_800.jpg')
+            radii_planets = {'Titan': 2575e3,
+                             'Europa': 1561e3,
+                             'Enceladus': 252e3,
+                             'Ganymede': 2631e3,
+                             'Mars': 3389.5e3,
+                             'Venus': 6051.8e3}
```

Comment:
For Venus it would still plot the continents as this branch is only found if the radius is smaller than 5000 km.